src/main/java/com/google/api/codegen/util/go/GoTypeTable.java

```diff
@@ -104,4 +105,38 @@ private static boolean isStandardImport(String importPath) {
     // we might have to special case them.
     return !importPath.contains("/");
   }
+
+  /**
+   * A set of Go reserved words. See
+   * https://golang.org/ref/spec#Keywords
+   */
+  public static final ImmutableSet<String> KEYWORD_BUILT_IN_SET =
```

Comment:
Any chance we can call this `RESERVED_KEYWORD_SET`?
I know `KEYWORD_BUILT_IN_SET` is the same naming as in `NodeJSTypeTable`, but I think the above name is a bit more descriptive for this set.

I wonder if it's having another set or including non-reserved identifiers like type names. While we technically don't have to, we could run into some hairy situations like:
```
int := 3
```